docs/UserGuide.md

```diff
@@ -262,12 +263,10 @@ Example:
 Finds the deliverables whose titles or descriptions contain any of the given keywords.
 
 Format: `find KEYWORDS`
-* `KEYWORDS` contains one or more keywords used to match deliverables.
-* Searches only consider title and description.
-* Searches are case-insensitive, e.g. `homepage` will match `Homepage`.
-* Order of keywords does not matter, e.g. `Homepage Navigation` will match `Navigation Homepage`.
-* Searches only account for full words, e.g. `Deploy` will not match `Deployment` but `Stand-up` will match `Up button` and `Laptop stand`. 
-* Searches return deliverables matching at least one keyword, e.g. `Homepage Navigation` will return `Complete Homepage` and `Increase size of Navigation Bar`.
+
+Parameter      | Description                   | Example 
+---------------|-------------------------------|--------------------------------
+`KEYWORDS`     | One or more keywords used to match the deliverables to be found. <br><br>Details:<br>- Searches only consider title and description. <br>- Searches are case-insensitive, e.g. `homepage` will match `Homepage`.| `Homepage Navigation`: represents `Complete Homepage` and `Increase size of Navigation Bar` among others
```

Comment:
I think for description it can be: "One or more keywords used to search for the deliverables you need." Details: "Only searches based on title and description of deliverables." For the egs, "... : can match phrases such as ..."